src/utils/Types.tsx

```diff
@@ -95,9 +94,9 @@ export interface PlaceIconProps {
 
 export interface state {
   rows: row[];
-  combinedRows: combinedRow[];
+  allCombinedRows: combinedRow[];
   tooltipContent: string;
-  currentCombinedRow: combinedRow;
+  currentCombinedRows: combinedRow[];
```

Comment:
So you can have multiple combined rows selected at one time? You're going to have to explain that to me.

Reply:
The user can only ever select one combinedRow via clicking a map marker or table row, but I changed currentCombinedRow to currentCombinedRows due to how the markers split up/merge together upon zoom change. This allows for the highlight and details to persist among installations that split apart due to zooming in